plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java

```diff
@@ -66,9 +66,11 @@ public static boolean attemptTransfer(Group g, Player owner, UUID futureOwner) {
 					+ " is not a member of the group and can't be made primary owner!");
 			return false;
 		}
+		g.removeMember(futureOwner);
+		g.addMember(futureOwner, GroupManager.PlayerType.OWNER);
```

Comment:
Just thought I'd put this here :P 
![image](https://github.com/CivMC/Civ/assets/52572989/7a0df5b1-54b2-4cce-a2dc-45b44104cf6c)

Reply:
Best part about this is, I followed the logic of the promote player command... which doesnt even use this either

Reply:
Civ code D: